### ISSUE TITLE
[M0-Dev] Cómo van a estar relacionadas Empleado / Vacaciones con el calendario.

### ISSUE THREAD

**Issue by ignaciotitos** [RafaelT00/AutoIV#16]
En el [Milestone 0](https://github.com/RafaelT00/AutoIV/milestone/6)  y la #7 aparece la necesidad de un calendario para saber los días asignados, ¿Cómo van a estar relacionados estas estructuras? Ya que propones que calendario tiene que ser una clase aparte,  que aparezca el empleado los días en el calendario que tiene aplicados es como lo entiendo. Tu me dices.

**Comment by RafaelT00** [comment]
Sí, pensé el calendario como una clase para almacenar las asignaciones y además así tener un seguimiento. La clase Vacaciones o AsignarVacaciones creo que cumplen bien esto.

**Comment by ignaciotitos** [comment]
Vale perfecto.

**Comment by danielsp13** [comment]
Todos los issues necesitan estar asociado a una de las Historias de Usuario del proyecto. Así que, en el cuerpo del issue, especifica que este problema lo has extraido del análisis de dicha HU. Si no, no es posible validar el código que hagas.

**Comment by ignaciotitos** [comment]
Hecho! Gracias!

**Comment by JJ** [comment]
Por favor, consultad las diferentes sesiones y los errores frecuentes en este objetivo. Los issues no son un intercambio entre las dos personas, sino un planteamiento de un problema, que se tiene que resolver en código. Os lo ha indicado @danielsp13 , sería conveniente que le hicierais caso. Yo en este issue no veo ningún problema planteado, y menos una solución en el código que se refiere a esto.

**Comment by ignaciotitos** [comment]
Ok. Cierro el issue.

**Comment by JJ** [comment]
Lo he dicho varias veces, pero lo repetiré. **No se pueden cerrar issues desde el interfaz. Siempre desde un commit**. Si no, @RafaelT00 tendrá error en sus tests siempre a partir de ahora.

**Comment by ignaciotitos** [comment]
Perdón. ¿Debería de volver a abrirlo o lo dejo así?

**Comment by JJ** [comment]
> Lo he dicho varias veces, pero lo repetiré. **No se pueden cerrar issues desde el interfaz. Siempre desde un commit**. Si no, @RafaelT00 tendrá error en sus tests siempre a partir de ahora.

¿Se te ocurre otra forma de solucionarlo que no sea abrirlo y cerrarlo correctamente?

**Comment by ignaciotitos** [comment]
La verdad que no.

**Comment by ignaciotitos** [comment]
Supongo que poniendo la etiqueta invalid como se ha hecho anteriormente, pero no lo se.

**Comment by JJ** [comment]
Puedes poner la etiqueta que quieras. El tema es que tienes que volver a abrirlo y cerrarlo desde un commit, como cualquier otro issue.

**Comment by ignaciotitos** [comment]
Okey. Así lo haré.
